sync2jira/downstream_issue.py

```diff
@@ -960,12 +960,24 @@ def _update_github_project_fields(client, existing, issue, github_project_fields
     """
 
     for name, values in github_project_fields.items():
-        _, jirafieldname = values
-        try:
-            existing.update({jirafieldname: str(getattr(issue, name))})
-        except JIRAError:
-            # Add a comment to indicate there was an issue
-            client.add_comment(existing, f"Error updating GitHub project field")
+        fieldvalue = str(getattr(issue, name))
+        if not fieldvalue:
+            continue
```

Comment:
I don't think that this is going to do what you want it to:  converting the `getattr()` return to a `str` will produce `"None"` when the attribute is not present, and that value is _not_ "false-y".

Why are you coercing the value to be a string?  I would think it would normally be a string (without conversion), and otherwise you'll trap it with the `if` statement.  So, if you just remove the `str()` conversion, then the code should work.